frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -135,6 +135,7 @@ local order = {
         "terminal",
         "----------------------------",
         "plugin_management",
+        "patchmanager",
```

Comment:
Shouldn't that be patch_manager to be consistent with the other menu entries? ^_^

Reply:
If we want that to be ultimate consistant , it should be "Patch management".
In the code and in the menu.

Reply:
Yes, it should also be management, good point. But the _ thing jumps out without even looking at the words. ;-)